WalletWasabi.Fluent/Controls/Dialog.xaml

```diff
@@ -0,0 +1,61 @@
+<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:c="clr-namespace:WalletWasabi.Fluent.Controls">
+  <Design.PreviewWith> 
+  </Design.PreviewWith>
+  <Styles.Resources>
+  </Styles.Resources>
+ <Style Selector="c|Dialog">
+    <Setter Property="Template">
+      <ControlTemplate>
+        <Panel>
+          <Panel Background="#272727" Name="PART_Overlay">
+            <Button Name="PART_OverlayButton" Classes="invisible" />
```

Comment:
Get rid of this, we dont need a button, PART_Overlay.PointerReleased or Pressed.